app/controllers/todos_controller.rb

```diff
@@ -43,16 +45,25 @@ def destroy
 
   private
 
+  # Set the TodoList by user_id and todo_list_id
   def set_todo_list
     @todo_list = TodoList.find_by(id: params[:todo_list_id], user_id: params[:user_id])
-    return render json: { error: 'Todo list not found' }, status: :not_found unless @todo_list
+    unless @todo_list
+      logger.error "Todo list not found: #{params[:todo_list_id]}"
```

Comment:
is this logger directly accessible?

Reply:
No, logger doesn’t send logs anywhere outside by default as it will write within the file or showcases into the terminal when we run the rails server.its used for debugging the error so its not needed as of now.